platform/viewer/src/routes/routesUtil.js

```diff
@@ -19,7 +19,9 @@ const StudyListRouting = asyncComponent(() =>
   )
 );
 const StandaloneRouting = asyncComponent(() =>
-  import(/* webpackChunkName: "ConnectedStandaloneRouting" */ '../connectedComponents/ConnectedStandaloneRouting.js')
```

Comment:
Unnecessary change you can remove from this PR. We can lint all files later

Reply:
Yeah this freaking automatic linter keeps surprising me... I'll remove it. Thanks for that.